### ISSUE TITLE
Homekit Bridge not found

### ISSUE THREAD

**Issue by gfacek** [jensweigele/ioBroker.yahka#387]
Hi,

last weekend I tried to connect a camera to Homekit via yahka. This actually worked, but the playback was quite slow, which was due to the low performance of my iobroker VM. So I gave up on that and removed the camera in the yahka adapter again. Either immediately or shortly thereafter, controlling my devices via Homekit stopped working ("No response" on devices).

I have now already tried a few things. Unfortunately, the first thing I tried was to remove the bridge from Homekit and add it again. Homekit now does not find it anymore. Among other things, I have subsequently updated to version 17 of the adapter, downgraded to version 11, enabled "Use legacy Advertiser" after updating to 13.1, rebooted umpteen times (both iobroker and the entire VM), renamed the bridge, changed the MAC address. Nothing helped.

Now I am at the end with my ideas and google search and hope for help here.

Thanks in advance and best regards
Tobias

**Comment by gfacek** [comment]
Very strange! Immediately after restarting the adapter I see the bridge in Homekit when searching for new devices. However, I can not set it up. I can not enter the PIN in the form that is provided in the configuration of the bridge in the adapter (XXX-XX-XXX). The PIN is then displayed in the form of XXXX-XXXX. It then takes 1-2 minutes and terminates with an error.
If I wait too long, I also no longer see the bridge in the search.

**Comment by gfacek** [comment]
This is crazy! The webinterface has hung up. After I restarted the VM, the "trick" no longer works.

**Comment by gfacek** [comment]
I have just installed the Homekit Controller Adapter from Ingo Fischer. In this I can find the yahka bridge and connect to it without a problem.

**Comment by do1osc** [comment]
Same problem here.

**Comment by adieball** [comment]
Same here

**Comment by DerUntote** [comment]
same here!

**Comment by Juweii** [comment]
Me too

**Comment by gfacek** [comment]
I just installed a new instance of iobroker on a remaining Raspberry Pi and have installed the yahka adapter there as well. Of course hoping that the bridge won't be found there either. But, guess what...this one was found in Homekit and I can add and use it as well.
So there must be an error in the configuration. But I honestly do not want to reconfigure all devices. Most are quite easy to configure, but for some I had to set up a conversion for color, for example.
But shouldn't it result in the same thing if I install a second instance of yahka?

**Comment by Juweii** [comment]
I found the issue on my installation. Since it was several months ago I have used HomeKit and so Yahka, I didn't notice the default change to avahi. In my ioBroker docker (buanet) command I needed to add the `--env AVAHI=true` and it worked.

**Comment by gfacek** [comment]
I've been running iobroker as a standalone VM for 3 years, as I had some issues with the Docker container at the time. But still your mention of avahi gave me an idea that I want to verify first. Maybe it has not directly to do with the integration of a camera, but with the installation of ffmpeg. 
I don't know right now if the bridge of my test installation is only not displayed anymore because I removed it from Homekit, or because I also installed ffmpeg there. I'm reinstalling iobroker on the spare RPi right now :)

**Comment by gfacek** [comment]
Crap, that was not it. It was probably just a coincidence.

Since I was still not satisfied with the performance of iobroker, I now quickly (~6 hours of work) installed an Intel NUC with Ubuntu Server and iobroker, and transferred my configuration. I unfortunately had to reconnect all zigbee devices as well, but at least my scripts and yahka configuration still worked. Now yahka works correctly again and also my cameras run properly with Homekit.

**Comment by do1osc** [comment]
@gfacek 
do you use ios16 ? with new PIN-format ?

**Comment by gfacek** [comment]
Yes, I'm every device on iOS 16. I entered my PIN in yahka like in the example 123-45-678 and entered it in Homekit without the dashes (12345678). Homekit inserted one Dash automatically (1234-5678). Worked fine!
